README.md

```diff
@@ -1,83 +1,92 @@
-2. Navigate to rover-ros2 workspace
-    * `cd rover-ros2`
-3. Source the workspace
-    * `source install/setup.bash`
-4. Launch ANCHOR
-    * `ros2 launch rover_launch.py mode:=anchor`
-
-## Launching as Standalone
-
-For use when running independent modules through their respective computers (pi/NUC) without ANCHOR.
-
-1. SSH to the the module's computer
-    * Core1: `ssh clucky@192.168.1.69`
-    * Core2: `ssh clucky@192.168.1.70`
-    * Arm: `ssh arm@192.168.1.70`
-    * Bio: \<TBD>
-    * Password: \<can be found in the rover-Docs repo under documentation>
-2. Run the main node (this sends commands to the MCU)
-    * Navigate to the rover-ros2 workspace (location may vary)
-        * `cd rover-ros2`
-    * Source the workspace
-        * `source install/setup.bash`
-    * Start the node
-        * ARM: `ros2 launch rover_launch.py mode:=arm`
-        * CORE: `ros2 launch rover_launch.py mode:=core`
-        * BIO: `ros2 launch rover_launch.py mode:=bio`
-
-## Running Headless
-
-Headless control nodes (for ARM and CORE) allow running of the module on-rover without the operator having ROS2 installed on their machine. You will need a laptop to connect to the pi/NUC in order to launch headless but it can be disconnected after the nodes are spun up.
-<br>
-1. SSH to the the module's computer
-    * Core1: `ssh clucky@192.168.1.69`
-    * Core2: `ssh clucky@192.168.1.70`
-    * Arm: `ssh arm@192.168.1.70`
-    * Password: \<can be found in the rover-Docs repo under documentation>
-2. Run the  headless node
-    * You must have ANCHOR or the module's Standalone node running
-    * Open a new terminal (SSH'd to the module)
-    * Navigate to rover-ros2 workspace
-        * `cd rover-ros2`
-    * Source the workspace
-        * `source install/setup.bash`
-    * Run the node (ensure controller is connected and on x-input mode)
-        * CORE: `ros2 run core_pkg headless`
-        * ARM: `ros2 run arm_pkg headless`
+## About
+
+Includes all ROS2 packages for the rover. These are centrally located for modular rover operation.
+
+You will use this package to launch all module-side ROS2 nodes.
+
+# Table of Contents
+
+1. Title
+3. Table of Contents
+3. Software Requirements
+    1. ROS2 Humble
+    2. IP Configuration
+4. Hardware Requirements
+5. Recommended Programs
+6. How to Use
+7. Common Problems/Troubleshooting
+8. Major To-Do Items
+9. Author(s) 
+10. Maintainer(s)
+
+# Software Requirements 
+
+Since July/August of 2025, ASTRA uses NixOS. A functioning NixOS installation is assumed.
+
+## ROS2 Humble
+
+We use [ROS2 Humble](https://docs.ros.org/en/humble/index.html), and plan on using that until support for it expires. 
+
+To install ROS2, run the following command in your /etc/nixos folder:
+```nix
+nix flake init --template github:lopsided98/nix-ros-overlay
+```
+
+After you install ROS2, you can install all dependencies needed by this repository by running the following commands:
+
+```bash
+$ sudo rosdep init  # only run once
```

Comment:
this won't work under nixos, so it's strange having this right below our nixos instructions. we can work on the new readme together